cli/src/main/java/Main.java

```diff
@@ -20,7 +20,12 @@ public static void startScreen() {
             System.out.println("2. Play");
             System.out.println("3. Quit");
             
-            input = Integer.parseInt(scan.nextLine());
+            while (!scan.hasNextInt()) {
```

Comment:
It could be called something like `getIntFromUser` and doesn't return until they enter an integer